go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/brancz/gojsontoyaml v0.1.0
 	github.com/campoy/embedmd v1.0.0
 	github.com/dgryski/go-jump v0.0.0-20211018200510-ba001c3ffce0
+	github.com/fsnotify/fsnotify v1.5.4
```

Comment:
Can we use a later version here (v1.6.0+)?
```
inotify: replace epoll() with non-blocking inotify (https://github.com/fsnotify/fsnotify/pull/434)

Non-blocking inotify was not generally available at the time this library was written in 2014, but now it is. As a result, the minimum Linux version is bumped from 2.6.27 to 2.6.32. This hugely simplifies the code and is faster.```

See: https://github.com/fsnotify/fsnotify/releases/tag/v1.6.0